par-benchmarks/util/ForkJoinMPL.sml

```diff
@@ -0,0 +1,15 @@
+structure ForkJoin : FORK_JOIN = struct
+
+(* MPL's builtin ForkJoin. *)
+structure FJ = ForkJoin
+
+val par = FJ.par
+val parfor = FJ.parfor
+val pmap = List.map
+
+fun alloc n _ = Array.alloc n
+fun pair (f,g) (x,y) = par (fn () => f x, fn () => g y)
+
+type gcs = int * int
+fun parfor' _ = parfor 0
```

Comment:
We could write

    fun parfor' ((_,g):gcs) = parfor g

Reply:
Oh yeah, that's probably better, I'll change it...

Reply:
This was also the reason why MPL wouldn't run most of the bencmarks.  Much better now!